server/settings/publish_plugins.py

```diff
@@ -1171,6 +1248,13 @@ class PublishPuginsModel(BaseSettingsModel):
             },
         ]
     },
+    "CollectExplicitResolution": {
+        "enabled": True,
+        "product_types": [
+            "shot"
+        ],
```

Comment:
Note that having this empty by default but enabled... just means that we're running more logic (`get_attr_defs_for_instance`) by default, for ALL instances... regardless of whether `product_types` is defined.

Should this maybe have been disabled by default?